uint8[255] message
---
uint8[255] response
